src/ui/Box/Box.tsx

```diff
@@ -0,0 +1,8 @@
+import { styled } from '../../stitches.config';
+
+export const Box = styled('div', {
```

Comment:
I'm not a fan of having this here, ideally I think we'd have a style reset included as css for the whole app

Reply:
This is how stitches recommended to do, but If want we can move it into a global css